templates/user/edit-profile.html

```diff
@@ -269,6 +347,23 @@
                                 </a>
                             </td></tr>
                         {% endif %}
+                        {% if profile.is_totp_enabled %}
```

Comment:
Assuming the screenshots from https://github.com/DMOJ/online-judge/pull/1334#issuecomment-626259738 are still correct, scratch codes should probably be moved under `Two-factor authentication` but before the API tokens/security keys since they are closely intertwined.